packages/nbdime/package.json

```diff
@@ -19,26 +19,32 @@
     "watch": "tsc --build --watch"
   },
   "dependencies": {
-    "@jupyterlab/codeeditor": "^2 || ^3",
-    "@jupyterlab/codemirror": "^2 || ^3",
-    "@jupyterlab/coreutils": "^4 || ^5",
-    "@jupyterlab/nbformat": "^2 || ^3",
-    "@jupyterlab/outputarea": "^2 || ^3",
-    "@jupyterlab/rendermime": "^2 || ^3",
-    "@jupyterlab/services": "^5 || ^6",
-    "@lumino/algorithm": "^1.1.2",
-    "@lumino/coreutils": "^1.3.0",
-    "@lumino/dragdrop": "^1.3.0",
-    "@lumino/signaling": "^1.2.2",
-    "@lumino/widgets": "^1.6.0",
+    "@codemirror/lang-markdown": "^6.1.1",
+    "@codemirror/language": "^6.6.0",
+    "@codemirror/legacy-modes": "^6.3.2",
+    "@codemirror/state": "^6.2.0",
+    "@codemirror/view": "^6.14.0",
+    "@jupyterlab/apputils": "^4.0.0",
```

Comment:
Also here dev deps became runtime deps. Intentional?

Reply:
This will be removed as it was previously for the common dependencies.